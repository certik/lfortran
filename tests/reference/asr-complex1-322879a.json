{
    "basename": "asr-complex1-322879a",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/complex1.f90",
    "infile_hash": "3243c29c947c9700891971aae8780afac25cd01387e0307063f3928a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-complex1-322879a.stdout",
    "stdout_hash": "abf5b200287e541ddba374057f68462478cb2dd5838efdb240361127",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}