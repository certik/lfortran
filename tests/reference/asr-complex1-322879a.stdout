(TranslationUnit (SymbolTable 1 {complex1: (Program (SymbolTable 2 {x: (Variable 2 x Local () () Default (Complex 4 []) Source Public Required .false.)}) complex1 [] [(= (Var 2 x) (ComplexConstructor (RealConstant   3.00000000000000000e+00 (Real 4 [])) (RealConstant   4.00000000000000000e+00 (Real 4 [])) (Complex 4 []) (ComplexConstant   3.00000000000000000e+00   4.00000000000000000e+00 (Complex 4 []))) ())])}) [])
